.github/workflows/certora.yml

```diff
@@ -0,0 +1,46 @@
+name: Certora
+
+on: [push, pull_request]
```

Comment:
maybe trigger the Certora action only on pull requests as it takes some time to run?